music21/midi/__init__.py

```diff
@@ -715,7 +720,7 @@ def setPitchBend(self, cents, bendRange=2):
         self.parameter1 = d2
         self.parameter2 = d1  # d1 is most significant byte here
 
-    def parseChannelVoiceMessage(self, midiBytes: bytes) -> bytes:
+    def parseChannelVoiceMessage(self, byte0: int, midiBytes: bytes) -> bytes:
         r'''
         Take a set of bytes that represent a ChannelVoiceMessage and set the
         appropriate enumeration value and data, returning the remaining bytes.
```

Comment:
```suggestion
        appropriate enumeration value and data, returning the remaining bytes.

        Changed in v7.3 -- a description of the changes (we don't have sphinx versionchanged blocks yet, just some text)
```

Reply:
I'll write here rather than committing. If it seems good, I'll add a commit.

```
Changed in v7.3 -- the status byte is now accepted as the parameter `byte0`. Previously the first byte of `midiBytes` was expected to carry the status. Previous version calls to `MidiEvent.parseChannelVoiceMessage(midiBytes)` should be replaced with `MidiEvent.parseChannelVoiceMessage(midiBytes[0], midiBytes[1:])`
```

Reply:
Alternatively, I can add `byte0` as an optional last parameter, and have `parseChannelVoiceMessage()` read it from `midiBytes` if it isn't specified. That way it won't be a breaking change.

Reply:
...and that's what I did.

Reply:
Fantastic, I was also going to suggest that as an option. I think a changed notice explaining when the new parameter was added would still be nice. We don't have different sets of docs by version, so someone could be reading this and still be on 5.7.2...

`Changed in v.7.3 -- Added optional `byte0` argument for better performance.`